code/game/objects/structures/crates_lockers/crates/critter.dm

```diff
@@ -41,10 +41,7 @@
 		. += "manifest"
 
 /obj/structure/closet/crate/critter/return_air()
-	if(tank)
-		return tank.air_contents
-	else
-		return loc.return_air()
+	return tank ? tank.air_contents : ..()
```

Comment:
This changes behavior

Reply:
I mean, it would, but parent is `loc?.return_air()` so it works out.